homeassistant/components/thermostat/__init__.py

```diff
@@ -301,22 +306,32 @@ def turn_fan_off(self):
     @property
     def min_temp(self):
         """Return the minimum temperature."""
-        return convert(7, TEMP_CELSIUS, self.unit_of_measurement)
+        unit = self.unit_of_measurement
+        if not unit:
```

Comment:
If an implementation does not define a unit_of_measurement, [`we blow up`](https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/components/thermostat/__init__.py#L237-L240)